File: CSharp/1940.cs
using System;
using System.Linq;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {   
        int[] n = Console.ReadLine().Split().Select(int.Parse).ToArray();

        List<int> pontos = new List<int>();
        for(int i=0;i<n[0];i++)
        {pontos.Add(0);}
        int[] listao = Console.ReadLine().Split().Select(int.Parse).ToArray();

        for(int i =0;i<n[1];i++)
        {
            for(int j =0;j<n[0];j++)
            {
                pontos[j]+=listao[j+i*n[0]];
            }
        }
        Console.WriteLine(pontos.LastIndexOf(pontos.Max())+1);
    }
}

File: Python/1940.py
n=list(map(int,input().split()))
pontos=list()
for i in range(n[0]):
    pontos.append(0)



lista= list(map(int,input().split()))

for i in range(n[1]):
    for j in range(n[0]):
        pontos[j]+=lista[j+i*n[0]]
maximo=max(pontos)

#for i in range(len(pontos)-1,-1):
for i in range(len(pontos)-1,-1,-1):
    if pontos[i]==maximo:
        print(i+1)
        break